bot/src/main/java/com/community/tools/service/github/DiscordGithubService.java

```diff
@@ -0,0 +1,20 @@
+package com.community.tools.service.github;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import org.apache.logging.log4j.util.Strings;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DiscordGithubService {
+
+  public Map<String, String> getDiscordGithubUsernames(List<String> githubUsernames) {
+    return Collections.emptyMap();
```

Comment:
What prevents us from implementing this logic right now?

Reply:
This class is part of the 'Create Discord registration' task. I created a mock class for my part so that the above task does not block the current task from executing. This method interface is provided by another trainee.
This method is an empty method. It will be implemented in the 'Create Discord registration' task.

Reply:
gotcha 👌